packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -286,13 +286,19 @@ class ProxyController {
             return;
         }
 
-        let responseData = '';
+        const responseData: Buffer[] = [];
+        let responseLen = 0;
 
         responseStream.data.on('data', (chunk: Buffer) => {
-            responseData += chunk.toString();
+            responseData.push(chunk);
+            responseLen += chunk.length;
         });
 
         responseStream.data.on('end', async () => {
+            if (responseLen > 50_000_000) {
```

Comment:
i would log everything > 5-10mb to have a better idea. I would be surprised if we have a lot of those but maybe I am wrong

Reply:
Cool cool, I waffled on the issue but based on what I saw we probably have to have someone doing something massive for this to be the cause of our issue.

Reply:
Fixed in dd759bf